dashboard/app/controllers/featured_projects_controller.rb

```diff
@@ -1,6 +1,14 @@
 class FeaturedProjectsController < ApplicationController
   authorize_resource
 
+  def bookmark
+    _, project_id = storage_decrypt_channel_id(params[:project_id])
+    return render_404 unless project_id
+    @featured_project = FeaturedProject.find_or_create_by!(project_id: project_id)
+    @featured_project.update! unfeatured_at: nil, featured_at: nil
+    buffer_abuse_score
```

Comment:
Outside the scope of this PR, but I'm curious what the plan is for abuse scores. Can students still report abuse on bookmarked projects?

Reply:
Great question! Just tagging @samantha-code on this. We're in the middle of discussion about how users will submit consent to be featured in gallery so I'll add this to list of questions that are being worked out. Thanks!

Reply:
https://docs.google.com/document/d/1vU68tzXG72Z80MvM6xYEDwF5hLDf7V097P-0dDfpouc/edit?disco=AAABHLofTvY